bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdisplay.py

```diff
@@ -0,0 +1,24 @@
+# Author: Giacomo Parolini
+
+################################################################################
+# Copyright (C) 1995-2022, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from . import pythonization
+
+@pythonization("RDisplay", ns="ROOT::RDF")
+def pythonize_rdisplay(klass):
+    """
+    Parameters:
+    klass: class to be pythonized
+    """
+    def repr(klass):
+        import ROOT
+        opts = ROOT.RDF.RDisplay.RPrintOptions()
+        opts.fFormat = ROOT.RDF.RDisplay.EPrintFormat.kHtml
+        return klass.AsString(opts)
+    klass._repr_html_ = repr
```

Comment:
Maybe we can add for the non-IPython interactive use case
```
klass.__repr__ = lambda self: self.AsString()
```

which will show the markdown table which is friendlier for terminal-based interaction